wls-admin-service/src/test/java/de/muenchen/oss/wahllokalsystem/adminservice/client/infomanagement/KonfigurierterWahltagClientImplTest.java

```diff
@@ -1,12 +1,14 @@
 package de.muenchen.oss.wahllokalsystem.adminservice.client.infomanagement;
 
+import de.muenchen.oss.wahllokalsystem.adminservice.eai.basisdaten.client.WahlenControllerApi;
 import de.muenchen.oss.wahllokalsystem.adminservice.eai.infomanagement.client.KonfigurierterWahltagControllerApi;
```

Comment:
Da die Objektvariable unused ist.

```suggestion

import de.muenchen.oss.wahllokalsystem.adminservice.eai.infomanagement.client.KonfigurierterWahltagControllerApi;
```